meson.build

```
@@ -163,7 +165,27 @@ pico = custom_target(
     'pico',
     input: gekkoboot_dol,
     output: 'gekkoboot_pico.uf2',
-    command: [dol2ipl, '@OUTPUT@', '@INPUT@'],
+    command: [dol2ipl, '@OUTPUT@', '@INPUT@', 'rp2040'],
+    build_by_default: true,
+    install: false,
+    install_dir: '/',
+)
+
+pico2 = custom_target(
+    'pico2',
+    input: gekkoboot_dol,
+    output: 'gekkoboot_pico2.uf2',
+    command: [dol2ipl, '@OUTPUT@', '@INPUT@', 'rp2350'],
+    build_by_default: true,
+    install: false,
+    install_dir: '/',
+)
+
+pico_universal = custom_target(
+    'pico_universal',
+    input: [pico, pico2],
+    output: 'gekkoboot_universal.uf2',
+    command: ['sh', '-c', 'cat "$1" "$2" > "$3"', 'sh', '@INPUT0@', '@INPUT1@', '@OUTPUT@'],
```

The redirect operator didn't work because Meson doesn't run build commands in a shell. Anything involving shells is woefully unportable so they should be avoided at all costs.

The correct solution here is to use [`capture`](https://mesonbuild.com/Reference-manual_functions.html#custom_target_capture). See the packer build script for examples of how it's used:
https://github.com/redolution/gekkoboot/blob/c8f7e10d85864c9d3355190a7205d0c2412bb695/packer/meson.build#L44-L50

Try this:
```suggestion
    command: [cat, '@INPUT@'],
    capture: true,
```

`@INPUT@` means the entire list of inputs and it's safe to use here.

Also, use [`find_program()`](https://mesonbuild.com/Reference-manual_functions.html#find_program) instead of naming cat directly in your command:

https://github.com/redolution/gekkoboot/blob/c8f7e10d85864c9d3355190a7205d0c2412bb695/meson.build#L21-L22